ReplayBrowser/Pages/Search.razor

```diff
@@ -245,6 +236,7 @@
             NavigationManager.NavigateTo("/");
```

Comment:
(UNRELATED, but also should probably remove this line as well, idk why its there)